arches/app/models/models.py

```diff
@@ -809,6 +789,8 @@ def __init__(self, *args, **kwargs):
         on_delete=models.SET_NULL,
     )
 
+    objects = NodeQuerySet.as_manager()
```

Comment:
We discussed possibly moving the managers to a managers.py, but once I realized what I probably wanted to be doing was exposing the querysets and just linking them to the models with as_manager(), I felt much more comfortable starting a new querysets.py file 👍